socorro/unittest/external/postgresql/test_dbapi2_util.py

```diff
@@ -195,13 +196,14 @@ def __init__(self, name):
             things,
             [Description('first_name'), Description('last_name')]
         )
-        zipped = sequence.zipped()
+        zipped = sequence.zipped(wrapper=DotDict)
         ok_(isinstance(zipped, list))
         eq_(zipped[0], {'first_name': 'Peter', 'last_name': 'Bengtsson'})
         eq_(zipped[1], {'first_name': 'Lars', 'last_name': 'Lohn'})
 
         # Also, because the zipped list actually is made up for DotDict
         # instances you can access this like a proper ORM thing.
+        # This is thanks to us passing `wrapper=DotDict` above.
```

Comment:
What is the value of this? If in all real life cases you do not use ``DotDict``, then why not just remove that option?